src/components/header/styles/modal-account-unification.scss

```diff
@@ -0,0 +1,487 @@
+@import 'src/css/quasar.variables.scss';
+@import 'src/css/utils.scss';
+
+.wrapper--account-unification {
+  display: flex;
+  flex-direction: column;
+  gap: 48px;
+  @media (max-width: $sm) {
```

Comment:
Please fix all the 'max-width' to 'min-width' as we are using it on this repo.

Reply:
@impelcrypto fixed it ([15bec29](https://github.com/AstarNetwork/astar-apps/pull/935/commits/15bec290973a72bd4d5b9f22347180a99d9c7bbb))